pkg/netpol/connlist/parsed_resources_test.go

```diff
@@ -0,0 +1,1304 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+
+*/
+
+package connlist
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+	netv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/network-policy-api/apis/v1alpha1"
+
+	"github.com/np-guard/netpol-analyzer/pkg/internal/output"
+	"github.com/np-guard/netpol-analyzer/pkg/internal/testutils"
+	"github.com/np-guard/netpol-analyzer/pkg/manifests/parser"
+	"github.com/np-guard/netpol-analyzer/pkg/netpol/eval"
+)
+
+var (
+	udp        = v1.ProtocolUDP
+	serve80tcp = "serve-80-tcp"
+)
+
+func newDefaultPod(namespace, name string, ports []int, protocols []v1.Protocol) v1.Pod {
+	podObj := v1.Pod{}
+	podObj.TypeMeta.APIVersion = "v1"
+	podObj.TypeMeta.Kind = "Pod"
+	podObj.ObjectMeta.Name = name
+	podObj.ObjectMeta.Namespace = namespace
+	podObj.Labels = map[string]string{"pod": name}
+	for _, port := range ports {
+		for _, protocol := range protocols {
+			podObj.Spec.Containers = append(podObj.Spec.Containers, newDefaultContainer(port, protocol))
+		}
+	}
+	return podObj
+}
+
+func newDefaultContainer(port int, protocol v1.Protocol) v1.Container {
+	contObj := v1.Container{}
+	contObj.Name = fmt.Sprintf("cont-%d-%s", port, strings.ToLower(string(protocol)))
+	contObj.Ports = make([]v1.ContainerPort, 1)
+	contObj.Ports[0].Name = fmt.Sprintf("serve-%d-%s", port, strings.ToLower(string(protocol)))
+	contObj.Ports[0].ContainerPort = int32(port)
+	contObj.Ports[0].Protocol = protocol
+	return contObj
+}
+
+type podInfo struct {
```

Comment:
missing documentation, what this type captures/represetns

Reply:
Added documentation